docs/content/release-notes/flink-2.0.md

```diff
@@ -0,0 +1,1666 @@
+
+### MISC
+
+#### DataStream V2 API
+
+##### [FLINK-34547](https://issues.apache.org/jira/browse/FLINK-34547)
+
+The DataStream API is one of the two main APIs that Flink provides for writing data processing programs. As an API that was introduced practically since day-1 of the project and has been evolved for nearly a decade, we are observing more and more problems of it. Improvements on these problems require significant breaking changes, which makes in-place refactor impractical. Therefore, we propose to introduce a new set of APIs, the DataStream API V2, to gradually replace the original DataStream API.
+
+In Flink 2.0, we provide the MVP version of the new DataStream V2 API. It contains the low-level building blocks (DataStream, ProcessFunction, Partitioning), context and primitives like state, time service, watermark processing. At the same time, we also provide some high-level extensions, such as window and join. They are more like short-cuts / sugars, without which users can probably still achieve the same behavior by working with the fundamental APIs, but would be a lot easier with the builtin supports.
+
+See DataStream API(V2) [Documentation](https://nightlies.apache.org/flink/flink-docs-release-2.0/docs/dev/datastream-v2/overview/) for more details
+
+**NOTICE:** The new DataStream API is currently in the experimental stage and is not yet stable, thus not recommended for production usage at the moment.
+
+#### Java Supports
+
+##### [FLINK-33163](https://issues.apache.org/jira/browse/FLINK-33163)
+
+Starting the 2.0 version, Flink officially supports Java 21.
+
+The default and recommended Java version is changed to Java 17 (previously Java 11). This change mainly affect the docker images and building Flink from sources.
+
+Meanwhile, Java 8 is no longer supported.
+
+#### Serialization Improvements
+
+##### [FLINK-34037](https://issues.apache.org/jira/browse/FLINK-34037)
+
+Flink 2.0 introduces much more efficient built-in serializers for collection types (i.e., Map / List / Set), which is enabled by default.
+
+We have also upgraded Kryo to version 5.6, which is faster and more memory efficient, and has better supports for newer Java versions.
+
+# Highlight Breaking Changes
+
+## API
+
+The following sets of APIs have been completely removed.
+- **DataSet API** - Please migrate to DataStream API, or Table API/SQL if applicable. See also "How to Migrate from DataSet to DataStream".
+- **Scala DataStream and DataSet API** - Please migrate to the Java DataStream API.
+- **SourceFuction, SinkFunction and Sink V1** - Please migrate to Source and Sink V2.
+- **TableSoure and TableSink** - Please migrate to DynamicTableSource and DynamicTableSink. See also "User-defined Sources & Sinks".
+- **TableSchema, TableColumn and Types** - Please migrate to Schema, Column and DataTypes respectively.
+
+Some deprecated methods have been removed from DataStream API. See also the list of breaking programming APIs.
```

Comment:
Can we link to the list of the removed methods and the  list of breaking programming APIs